scripts/EN/Harry

```diff
@@ -4894,14 +4894,14 @@ LoadEffect(0x33, 0x1, 0x1, 0x2); // 0x6192: 0000406d
 DrawBackground(); // 0x619e: 00000065
 PlayMusic(0x71, 0x1); // 0x61a2: 0000206c
 LoadSecondEffect(0x33, 0x1, 0x1, 0x1); // 0x61aa: 0000406e
-ShowText((1499)"I speak. ¥"How about coming with me? This may not be the safest place in the world either. I can't promise you anything, but I'll do my best to protect you.¥""); // 0x61b6: 0000100c
+ShowText((1499)"I speak.¥nW2¥"How about coming with me?¥nW2This may not be the safest place in the world either. W1I can't promise you anything, but I'll do my best to protect you.¥""); // 0x61b6: 0000100c
 HandleInput(); // 0x61bc: 00000010
 LoadSecondEffect(0x33, 0x2, 0x1); // 0x61c0: 0000306e
-ShowText((1500)"¥"No...somehow I feel I'm not supposed to leave this place. Oh, Harry, I'm so scared... I'm cold.¥""); // 0x61ca: 0000100c
+ShowText((1500)"¥"W2No…W2somehow I feel I'm not supposed to leave this place.¥nW2Oh, Harry, W1I'm so scared…¥nW2I'm cold.¥""); // 0x61ca: 0000100c
 HandleInput(); // 0x61d0: 00000010
-ShowText((1501)"I waver. Is it ok for me to leave Lisa alone... But then, what will happen to Cheryl? Cheryl has only me to depend on. My concern for Cheryl cannot be compared with my concern for Lisa. After all...she is my daughter."); // 0x61d4: 0000100c
+ShowText((1501)"I waver. W2Is it okay for me to leave Lisa alone?¥nW2But then, W1what will happen to¥nW2Cheryl?¥nW2Cheryl has only me to depend on. My concern for Cheryl cannot be compared with my concern for Lisa.¥nW3After all…she is my daughter."); // 0x61d4: 0000100c
```

Comment:
Changed ok to okay.

Also removed the ... from the second sentence and added a question mark. Just feels better that way.